src/azul/chalice.py

```diff
@@ -116,6 +121,21 @@ def patched_event_source_handler(self_, event, context):
         if old_handler.__code__ != patched_event_source_handler.__code__:
             chalice.app.EventSourceHandler.__call__ = patched_event_source_handler
 
+    def _wrapping_middleware(self, event, get_response):
```

Comment:
More descriptive name
```suggestion
    def _error_wrapping_middleware(self, event, get_response):
```